pkg/codegen/docs/templates/function.tmpl

```diff
@@ -19,7 +19,12 @@
 <div class="highlight"><pre class="chroma"><code class="language-go" data-lang="go"><span class="k">func </span>Lookup{{ .ResourceName }}<span class="p">(</span>{{ htmlSafe .FunctionArgs.go }}<span class="p">) (*{{ template "linkify_param" .FunctionResult.go }}, error)</span></code></pre></div>
 
 <!-- C# -->
-<div class="highlight"><pre class="chroma"><code class="language-csharp" data-lang="csharp"><span class="k">public static </span>Task<{{ template "linkify_param" .FunctionResult.csharp }}> <span class="p">Get{{ .ResourceName }}(</span>{{ htmlSafe .FunctionArgs.csharp }}<span class="p">)</span></code></pre></div>
+<div class="highlight">
+<pre class="chroma">
+<code class="language-csharp" data-lang="csharp"><span class="k">public static class </span><span class="nx">Get{{ .ResourceName }} </span><span class="p">{</span>
+    <span class="k">public static </span>Task<{{ template "linkify_param" .FunctionResult.csharp }}> <span class="p">InvokeAsync(</span>{{ htmlSafe .FunctionArgs.csharp }}<span class="p">)</span>
```

Comment:
I noticed the return type is fully qualified, for example:

http://content-bucket-b916193.s3-website-us-east-1.amazonaws.com/docs/reference/pkg/aws/s3/getbucket/

<img width="839" alt="Screen Shot 2020-03-20 at 12 54 11 PM" src="https://user-images.githubusercontent.com/710598/77201547-e7d90c00-6aa9-11ea-942b-5d0b69a09565.png">

Can we remove the namespace (`GetBucketResult` rather than `Pulumi.Aws.S3.GetBucketResult`) similar to what we do for the parameters?

Reply:
Done. The latest [commit](https://github.com/pulumi/docs/pull/2687/commits/3ab7964b5bfeaac0a78a4ddd4db87a553a4420ef) in the companion docs PR I have open, reflects this change.

Reply:
Oh also note that, I removed the namespace from property type names throughout as well. Let me know if you would like me to revert to showing the full namespace for the property type names, but just not for the invoke result.